tests/unit/test_schemas.py

```diff
@@ -6,70 +6,73 @@
 import tmt.utils
 
 PATH = os.path.dirname(os.path.realpath(__file__))
-SCHEMADIR = os.path.join(PATH, "../../tmt/schemas")
 ROOTDIR = os.path.join(PATH, "../..")
 
+TREE = tmt.Tree(ROOTDIR)
 
-@pytest.fixture
-def tests_schema():
-    return tmt.utils.load_schema('test.yaml')
 
+# This is what `Tree.{tests,plans,stories}`` do internally, but after getting
```

Comment:
Hmm, I am not sure, I thought default how's added are expected from tmt

@psss @lukaszachy can you pls chime in?

Reply:
Those will be handled by the "prenormalization".